offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/components/person/create/CreatePersonView.groovy

```diff
@@ -411,4 +408,13 @@ class CreatePersonView extends VerticalLayout {
         createPersonViewModel.outdatedPerson = null
 
     }
+
+    protected Optional<String> getOrganisation(Affiliation affiliation) {
+        Optional<String> foundOrganisation = Optional.empty()
+        createPersonViewModel.affiliationToOrganisations.each {
+            if(affiliation in it.value) foundOrganisation = Optional.of(it.key)
+        }
+
+        return foundOrganisation
+    }
```

Comment:
It is unclear to me what this method does. Finding the organization of affiliation should be as easy as calling `affiliation.organization`.

Reply:
```suggestion
    protected Optional<String> getOrganisation(Affiliation affiliation) {
        return affiliation?.organization in createPersonViewModel.affiliationToOrganisations ? Optional.of(affiliation.organization) : Optional.empty()
    }
```